FontLoader.js

```diff
@@ -160,6 +162,66 @@
 				adobeBlankDiv.style.fontFamily = FontLoader.referenceFontFamilies[0] + ", " + adobeBlankFallbackFont;
 			}
 		},
+		_processFonts: function(fonts) {
+			var self = this;
+			var processedFonts = [];
+
+			fonts.forEach(function (font) {
+				if (self._isValidFontObject(font)) {
+					processedFonts.push(font);
+				} else if (font.indexOf(':') > -1) {
```

Comment:
What if font is an object with family and weight but missing style?
`_isValidFontObject` will return `false` and then `indexOf` will throw an exception of 'TypeError: font.indexOf is not a function'.